settings.gradle

```diff
@@ -45,3 +45,20 @@ rootProject.children.each {
     it.name = it.name + "_" + scalaVersion
   }
 }
+
+/**
+ * Skips execution of all tasks for the project 'samza-test'.
+ * To include `samza-test`, initiate gradle build by setting the property 'includeSamzaTest'.
+ * To build with samza-test: './gradlew clean build -PincludeSamzaTest'
+ */
+def propertyName = "includeSamzaTest"
```

Comment:
Re 2. Clearly, we disagree, hence my original comment :) First, see comment below about whether we should exclude the whole project or not. Even if we decide to, I think the name is confusing for someone not familiar with the build & project structure. IMHO will be much clearer to indicate its function instead.